xknx/knxip/tpdu.py

```diff
@@ -0,0 +1,98 @@
+"""
+Created on 14.01.2021.
+
+@author: bonzius
+"""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from xknx.exceptions import UnsupportedCEMIMessage
+from xknx.telegram import GroupAddress, IndividualAddress, Telegram, TPDUType
+
+from .knxip_enum import CEMIMessageCode
+
+if TYPE_CHECKING:
+    # from typing import Union, List
```

Comment:
```suggestion
```

Reply:
`from Adress import InternalGroupAddress` this should also be removed. If it would be needed it had to be imported from `xknx.telegram.address`